### DISCUSSION TITLE
how to "exit quake mode"

### DISCUSSION THREAD

**Discussion by pedroangelini** [microsoft/terminal#14279] [Ideas]
I use Quake mode a lot, but sometimes you just want to detach that particular window from the top and use it as a normal window. 
If you have a normal window and name it "_quake", it will snap to the top and lose its tab bar and minimize/close controls.
I thought using the command palette and renaming it to something different than _quake would undo the process, but alas, it does not.
Should this be a feature? Or maybe should there be a command in the pallet to "un-quakerize" a window?

**Comment by leandrumartin** [comment]
A few years late, but I thought I'd comment the solution anyway; I found this thread while looking for the solution myself, and I figured somebody else might too.

In the command palette (ctrl+shift+p), you can select either the "Move tab to a new window" or "Move pane to a new window" options, and it will effectively un-quakerize the window. You can also map these to keyboard shortcuts in Settings.
